src/api/prop.js

```diff
@@ -42,3 +42,9 @@ export const string = create({
   deserialize: toNullOrString,
   serialize: toNullOrString
 });
+
+export const object = create({
+  default: {},
```

Comment:
Should be a function that returns an object (`() => ({})`). This currently would return the same object for all props using this.